cf-testd/cf-testd.c

```diff
@@ -312,7 +299,9 @@ static ServerConnectionState *CFTestD_NewConn(ConnectionInfo *info)
     struct sockaddr_storage addr;
     socklen_t size = sizeof(addr);
 
-    if (getsockname(ConnectionInfoSocket(info), (struct sockaddr *)&addr, &size) == -1)
+    if (getsockname(
+            ConnectionInfoSocket(info), (struct sockaddr *)&addr, &size) ==
+        -1)
```

Comment:
This is just gross.

Reply:
I think the original is gross as well ;)

Reply:
Hard to argue with that.

Reply:
similar to the other long line maybe something like this:
```
    if (getsockname(
            ConnectionInfoSocket(info),
            (struct sockaddr *)&addr, 
            &size) == -1)
    {
```

Reply:
Move ConnectionInfoSocket() and getsockname outside the if :)

Reply:
I don't know, I just like the original line the best.